pkg/api/peer_req_resp.go

```diff
@@ -15,7 +15,8 @@ type Peer struct {
 // PeerAddReq represents an incoming request to add a peer to the cluster
 type PeerAddReq struct {
 	Addresses []string          `json:"addresses"`
-	MetaData  map[string]string `json:"metadata"`
+	Zone      string            `json:"zone,omitempty"`
+	MetaData  map[string]string `json:"metadata,omitempty"`
```

Comment:
s/MetaData/Metadata

Reply:
@prashanthpai I would like to keep it like MetaData as it seems more appropriate to me

Reply:
@prashanthpai I have made this change.